model/status_diff.go

```diff
@@ -116,12 +117,9 @@ func StatusDiffBuilds(original, patch *build.Build) (BuildStatusDiff, error) {
 
 // getTestUrl returns the correct relative URL to a test log, given a
 // TestResult structure
-func getTestUrl(tr *task.TestResult) string {
-	if tr.URL != "" {
-		return tr.URL
-	}
-	if tr.LogId != "" {
-		return TestLogPath + tr.LogId
+func getTestUrl(tr *testresult.TestResult) string {
```

Comment:
I think I will leave this because this purposely returns the relative URL.

Reply:
Mmm, the other thing is if this is worth fixing since it is getting replaced with the new UI. @khelif96 does Spruce use this? It doesn't look like it from what I can tell.

Reply:
It does not use it but arguably this should use the function that everything else uses to remain consistent.

https://github.com/evergreen-ci/evergreen/blob/f3c337eb8504d80e33b2689aac82c64795c88ddd/model/testresult/testresult.go#L78